python/ray/rllib/models/model.py

```diff
@@ -21,8 +29,14 @@ class Model(object):
 
     def __init__(self, inputs, num_outputs, options):
         self.inputs = inputs
+        if options.get("free_logstd", False):
+            num_outputs = num_outputs // 2
```

Comment:
It seems like this option only makes sense if `num_outputs` is a multiple of 2, should we include an assert?